Vagrantfile.alternate.rb

```diff
@@ -0,0 +1,462 @@
+# -*- mode: ruby -*-
```

Comment:
I rewrote the Vagrantfile this way at first because it seemed like it made it more readable if you're trying to figure out "what's in these images" like I was. However it seems like it makes it much less readable if you're trying to figure out "what's in this specific image", so I went with the way it was previously written. 

It's not complete here but I figured I'd include it in case others do find it more readable in general